### DISCUSSION TITLE
rendering of devtools offset in host window - offscreen

### DISCUSSION THREAD

**Discussion by ordinaryorange** [cefsharp/CefSharp#3880] [General]
Version - nuget CefSharp.OffScreen.NETCore 95.7.141
Arch - tested on x86 and x64
.NET - 5.0
Operating Sys - Win 10
Tested with Minimal example, and cefClient.exe and problem not apparent in either.
Seems to be related to .NETCore version

Reproducible project [here](https://github.com/ordinaryorange/CefSharpTest) 

Here is a screenshot of the DevTools window
![image](https://user-images.githubusercontent.com/20490998/141405288-84eda330-9bce-4353-b5d6-7196f3804fbd.png)

**Comment by amaitland** [comment]
Converting to discussion as this is unlikely a bug.

Looks like you are probably using a high DPI display, you need to make your application DPI aware.

https://github.com/cefsharp/CefSharp/wiki/General-Usage#high-dpi-displayssupport

**Reply by ordinaryorange** [reply]
You are indeed correct. It is not a bug.
Added `Cef.EnableHighDPISupport();` and now it renders perfectly.

Thankyou for your assistance
